public/index.html

```diff
@@ -4,6 +4,9 @@
 <head>
     <meta charset="UTF-8">
     <title>PopKwiz</title>
+    <script src="https://kit.fontawesome.com/dbab727138.js" crossorigin="anonymous"></script>
```

Comment:
Any reason for FontAwesome over Material UI's icons?

Reply:
Ok i got rid of FontAwesome and used the material icons. I'm sure your concern is bundle size? Makes sense. I thought the fontawesome icons looked nicer but not worth the size. I did add an svg icon that i got from flaticons. Its the book with the lightbulb. 
![image](https://user-images.githubusercontent.com/35232697/107262349-0d466f80-6a06-11eb-9a7e-540c53a90833.png)

Reply:
Hahaha looks better is a valid reason too